src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -83,6 +83,12 @@ interface Props {
   SavedObjectFinder: React.ComponentType<any>;
   stateTransfer?: EmbeddableStateTransfer;
   hideHeader?: boolean;
+  // By default, embeddable.destroy() is called when this component unmounts.
+  // To prevent this default behavior, set this prop to true.
+  isDestroyPrevented?: boolean;
+  // Toggle off the border and shadow applied around the embeddable.
+  // By default, the embeddable will have a shadow and border around it.
+  isBorderless?: boolean;
```

Comment:
Hey @sikhote, why are these two changes necessary?

Reply:
In the process of checking creating an example of this in the `Examples` section, I discovered there is an existing `EmbeddableRenderer` that does the job. These changes are now removed.